### PR TITLE
Add secret operation in libvirt.py

### PR DESCRIPTION
**Pull request by lento-sun**
With this change, we can define a secret with create_secret()
and set base64 encoded password to it with set_secret_pwd()

Sign-offed by: Yi Sun(yisun@redhat.com)

### CODE REVIEW

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -3095,3 +3097,70 @@ def virsh_cmd_has_option(cmd, option, raise_skip=True):
...
+    try:
+        sec_uuid = re.findall(r".+\S+(\ +\S+)\ +.+\S+",
+                              ret.stdout)[0].lstrip()
+    except IndexError:
+        raise exceptions.TestError("Fail to get newly created secret uuid")
+
+    return sec_uuid
+
+
+def set_secret_pwd(sec_uuid, password):
```
Comment (balamuruhans) [inline virttest/utils_test/libvirt.py]: we can have the encoding part in `virsh.secret_set_value`
```
def secret_set_value(uuid, password, options=None, **dargs):
    if dargs.get("encode", False):
        password = base64.b64encode(password)
```

Comment (lento-sun) [inline virttest/utils_test/libvirt.py]: Thx for reviewing, but IMHO, the virsh command methods should be kept clean to do exactly what they can do. And "darags" should only contain valid virsh API keywords, such as debug/uri/session_id, etc, which could be found in virsh.command(). But definitely put the encoding in virsh.secret_set_value is much more convenient.
What do you think about it? @will-Do

Comment (balamuruhans) [inline virttest/utils_test/libvirt.py]: Agree, fine then let us have the param out of **dargs,
`secret_set_value(uuid, password, encode=False, options=None, **dargs)`

Comment (lento-sun) [inline virttest/utils_test/libvirt.py]: another PR submitted for this, thx for the advice
https://github.com/avocado-framework/avocado-vt/pull/1036

**Comment by lento-sun** [comment]
@balamuruhans pls have a review, I've removed the  "set_secret_pwd" method and did a modification in  https://github.com/avocado-framework/avocado-vt/pull/1036

**Comment by balamuruhans** [comment]
Thanks @lento-sun

**Pull request merged**